Research/QuantBook.cs

```diff
@@ -428,13 +428,13 @@ public dynamic GetFundamental(string ticker, string selector = null, DateTime? s
                 }
 
                 var optionFilterUniverse = new OptionFilterUniverse(option);
-                var distinctSymbols = allSymbols.Distinct();
+                var distinctSymbols = allSymbols.Distinct().Select(x => new OptionUniverse() { Symbol = x, Time = start});
```

Comment:
Think we should add a todo to cleaned this up a bit, for when we tackle futures, above we are using `OptionChainProvider.GetOptionContractList` which will basically a history request to the universe type and here we are recreating them again,